cmd/gaia/testnets/README.md

```diff
@@ -301,3 +301,18 @@ You should now see the unbonded tokens reflected in your balance and in your del
 gaiacli account <your_address>
 gaiacli stake delegation --address-delegator=<your_address> --address-candidate=<bonded_validator_address> --chain-id=<name_of_testnet_chain>
 ```
+
+### Submitting updates to the genesis file.
+
+From `gaia-6000` onward, we are initializing new testnets wtih delegated stake. In the near term, we intend to have updated tooling that allows folks in a starting account distribution to self delegate. In the meantime, here are some tips on submitting a manual pool requests. Merges will be granted on a case by case basis. No promises.
```

Comment:
Should we not make it clear that people should really be using the faucet? I don't think we want to confuse folks by suggesting that they submit a PR to this "momentary" genesis files which should really only be used to add the power from the previous testnet

Reply:
"manual pull requests"? And agreed with Rigel, I don't think we should encourage this.

Reply:
I mean some active community members asked. It's not fundamentally necessary.

Reply:
then we should have a big disclaimer that this is NOT the recommended approach, and link to how to use the faucet right at the beginning

Reply:
Are we planning on going through manually at the end of each testnet to set stake for the next genesis file? If not, people should be making PR's or we should change the faucet, since the faucet does not provide enough steak for someone to become an effective validator atm. (For sybil resistance purposes)

Reply:
No, we should be able to export the current distribution and validator set with `gaiad export` - agreed on more faucet steak though.